tests/pkg_yxml/Makefile.ci

```diff
@@ -0,0 +1,2 @@
+BOARD_INSUFFICIENT_MEMORY := \
```

Comment:
According to Murdock, there are some stm32 F0 based boards that should be added there: nucleo-f042k6, nucleo-f031k6 and stm32f030f4-demo

Reply:
done